### PR TITLE
Add Optimistic Kovan to NetworkSwitcher

### PR DESCRIPTION
**Pull request by platschi**
## Description
Simple check if user is on Optimism Kovan or Optimism Mainnet and display the network's name accordingly. 

To avoid confusion among current users, ``Optimism`` (Mainnet) label was not changed but when connected to Optimistic Kovan, the label changes visibly to ``OΞ Kovan``.

## Related issue
#39 

## Motivation and Context
#39

## How Has This Been Tested?
Tested with local dev environment & latest Chromium

## Screenshots (if appropriate)
Localhost with Metamask on Optimism Kovan:
Screenshot_2021-09-20 Kwenta

Localhost with Metamask on Optimism Mainnet:
Screenshot_2021-09-20 Kwenta(1)

### CODE REVIEW

**Comment by Admiral-REK** [comment]
Awesome work 👏 
Few things/suggestions from my side. (The first one is unrelated to your changes).

1. Looks like the automated checks are failing to run.  I assume there are some unset secrets in the repo, because `password: ${{ secrets.GH_PACKAGES_READ_ONLY }}` in `audit_build_verify.yml` seems to be evaluating to an empty string.
https://github.com/Kwenta/kwenta/pull/84/checks?check_run_id=3654453366

**(PR feedback starts here)**
2. If the automated checks ran, I think they might fail as there are a few things to fix in the PR according to the linter
```bash
➜  kwenta git:(fix/network-switch-kovan) npm run lint

> kwenta@2.45.2 lint
> eslint . --ext ts,.tsx && tsc


/Users/rek/Desktop/kwenta/sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx
  81:1   error    Delete `↹`                           prettier/prettier
  83:1   error    Delete `↹`                           prettier/prettier
  85:18  warning  Expected '===' and instead saw '=='  eqeqeq
  89:1   error    Delete `↹`                           prettier/prettier

✖ 4 problems (3 errors, 1 warning)
  3 errors and 0 warnings potentially fixable with the `--fix` option.

➜  kwenta git:(fix/network-switch-kovan) 

```
You can verify this locally by running: `npm run lint`.  Interestingly, `git` should have prevented you from making the commit via the pre-commit/husky script: 

```bash
➜  kwenta git:(fix/network-switch-kovan) git commit -am "Test unlinted commit"

> kwenta@2.45.2 lint
> eslint . --ext ts,.tsx && tsc


/Users/rek/Desktop/kwenta/sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx
  81:1   error    Delete `↹`                           prettier/prettier
  83:1   error    Delete `↹`                           prettier/prettier
  85:18  warning  Expected '===' and instead saw '=='  eqeqeq
  89:1   error    Delete `↹`                           prettier/prettier

✖ 4 problems (3 errors, 1 warning)
  3 errors and 0 warnings potentially fixable with the `--fix` option.

➜  kwenta git:(fix/network-switch-kovan) ✗ 

```

If you are using a pure text based editor, these things can be easy to miss, but running `npm run lint:fix`, or passing the `--fix` flag as in the above output will automate fixing a lot of the formatting related issues.  If you switch to something like Intellij, you'll get a visual of when the linter is not happy: 

Screenshot 2021-09-20 at 19 42 37

File: sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx [hunk 1]
```diff
@@ -78,7 +78,15 @@ const NetworksSwitcher: FC<NetworksSwitcherProps> = () => {
...
-
+	
+	const network = useRecoilValue(networkState);
+	
+	const getL2Network = () => {
+		if (network.id == 69) {
+			return `OΞ Kovan`;
+		} else return `Optimism`;
+	};
+
```
Comment (Admiral-REK) [inline sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx:89]: Suggestion:

By the time we get to this point in the function, we are mostly defining Components.  If we move this to the top of the function where the other values are assigned, and make use of the ternary operator, we can define it as as a one-liner. 

Note also: 
1. The string literals were changed to use single quotes in place of backticks to match the other string literals in the file.
2. As typeof `useRecoilValue(networkState).id` is `number`, I changed to strict equality to silence the lint warning.
3. Dropped the function wrapper and went for direct invokation as I didn't think the additional call was adding anything (I could be wrong)

```javascript
const NetworksSwitcher: FC<NetworksSwitcherProps> = () => {
	const { switchToL1, switchToL2 } = useNetworkSwitcher();
	const { t } = useTranslation();
	const isL2 = useRecoilValue(isL2State);
	const networkLabel = useRecoilValue(networkState).id === 69 ? 'OΞ Kovan' : 'Optimism';
	const theme = useTheme();
	const { blockExplorerInstance } = BlockExplorer.useContainer();

... then later ...

<L2Select
	formatOptionLabel={formatOptionLabel}
	controlHeight={28}
	options={OPTIMISM_OPTIONS}
	value={{ label: networkLabel, prefixIcon: 'Optimism' }}
...
```

I didn't include this, but you could also replace `69` with `NetworkId['Kovan-Ovm']` (will leave that call up to you/others) to remove the magic number/make the enum the source of truth.   However, it's fairly obvious in this case what 69 is referring to, and AFAIK that network ID won't change in the future.

Comment (platschi) [reply sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx:89]: Thanks a lot for your extensive feedback. Agree with changing to a one-liner at the top of the function, totally makes sense now that you mentioned it.

**Review by JChiaramonte7 — CHANGES_REQUESTED** [review]
Thanks @Admiral-REK and @platschi this PR is mostly good to go. The only thing I would change is to replace the "Optimism" & "OΞ Kovan" string literals with a translation string from [en.json](https://github.com/Kwenta/kwenta/blob/3d60a2df8defd779cb80dffb5d0a1041b4818d57/translations/en.json)

example:

https://github.com/Kwenta/kwenta/blob/3d60a2df8defd779cb80dffb5d0a1041b4818d57/sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx#L85

**Comment by Admiral-REK** [comment]
Trying to manually QA on the deployment, and looks like we are blocked by https://github.com/Synthetixio/js-monorepo/pull/80 plus an uptake of that release.

I'm seeing the same error referenced there in the console when I try to use the switcher.

**Comment by JChiaramonte7** [comment]
> Trying to manually QA on the deployment, but looks like we are blocked by [Synthetixio/js-monorepo#80](https://github.com/Synthetixio/js-monorepo/pull/80) plus an uptake of that release.
> 
> I'm seeing the same error referenced there in the console when I try to use the switcher.

Pull the latest from the dev branch, it should have the correct downgraded version of `@synthetixio/optimism-networks` for this to work for now.

**Review by JChiaramonte7 — APPROVED** [review]

**Pull request merged**